SECURITY.md

```diff
@@ -15,6 +15,7 @@ The first stable release of Apache Ozone is 1.0, the previous alpha and beta rel
 | 1.2.1         | :x:                |
 | 1.3.0         | :x:                |
 | 1.4.0         | :white_check_mark: |
```

Comment:
Is 1.4.0 still supported after 1.4.1?  How do we release a fix for 1.4.0?

Reply:
So we should only mark the `1.4.1` as "white_check_mark" right?
```suggestion
| 1.4.0         | :x: |
```

Reply:
Or we should delete this `SECURITY.md` file now, and put it to the website, in the future.

Reply:
https://github.com/apache/ozone-site/pull/102

Reply:
I think we should still keep `SECURITY.md`, but replace the `Supported Versions` table with a link to the website.